graphics/opengl/shader.cpp

```diff
@@ -155,7 +155,6 @@ GLuint Shader::createCompatShader(const char *shaderSource, GLenum shaderType, c
 	GLchar versionSource[20];
 	if (OpenGLContext.type == kContextGLES2) {
 		switch(compatGLSLVersion) {
-			case 100:
 			case 110:
 			case 120:
```

Comment:
According to this:
https://en.wikipedia.org/wiki/OpenGL_Shading_Language

OpenGL ES 2.0 is based on GLSL 1.20, so apart from GLSL version 1.0 (which is the internal hack that's removed by this PR), I believe we don't need version 1.10 either, since theoretically it shouldn't be feasible in the OpenGL ES 2.0 spec

Reply:
This value is provided by our code to indicate which version we should put in the header of the shader.
Our code provides 110 or 120 as a full-OpenGL version.
This avoids having to check which GL we use in several places of the code.

This piece of code remaps these two values on version 100 because GLES2 only knows about this version.